tests/Unit/Progress/Logger/BlinkenLoggerTest.php

```diff
@@ -161,6 +161,6 @@ public function testIterationEndStats()
         $this->variant->computeStats();
 
         $this->logger->variantEnd($this->variant);
-        $this->assertStringContainsString('RSD/r: 0.00%', $this->output->fetch());
+        $this->assertStringContainsString('±0.00%', $this->output->fetch());
```

Comment:
this is testing something for which this clsas is no longer responsible, same for other similar cases here.